### ISSUE TITLE
Get-AIDefaultProvider to balance the Set-AIDefaultProvider

### ISSUE THREAD

**Issue by dfinke** [dfinke/PSAI#52]

**Comment by Agazoth** [comment]
Get-AIProvider actually returns the default provider if no other parameters are set:
![{7B8E873F-7B61-4A17-AB71-2841BE2BACF2}](https://github.com/user-attachments/assets/88105871-0864-496c-be1a-f8600a298828)

**Comment by dfinke** [comment]
I'd prefer if `Get-AIProvider` worked like `dir`

`Get-AIProvider`, no param returns `-All`

and takes a wildcard:

`Get-AIProvider Open*`

Thoughts?

**Comment by Agazoth** [comment]
That could be done. -All would need to be replaced by a  Default switch or code has to be written to filter for the default in all cmdlets fetching the default provider, if no provider is provided.

`Get-AIProvider Open*`  can be done by looking for * in the name parameter and then return providers based on that filter.

Both can be done

**Comment by dfinke** [comment]
Yep. I need to look at fire up parts in the provider/model config. If I get time, I can take a look.

if($null -eq $name) { $name = '*' } 

then `-like` on the list
